app/src/main/java/com/eveningoutpost/dexdrip/EditAlertActivity.java

```diff
@@ -682,10 +685,9 @@ public void onTimeSet(TimePicker timePicker, int selectedHour, int selectedMinut
 
     }
 
-    private void chooseFile()
-    {
+    private void chooseFile() {
         final Intent fileIntent = new Intent();
-        fileIntent.setType("audio/mpeg3");
+        fileIntent.setType("*/*");
```

Comment:
Why do we allow all files here but filter it later? I thought the new file picker will handle it better.

Reply:
It is the same picker. Using it like this allows people to choose all files and later we have to filter them.
Not ideal, but does make life for users better. I did not find an easy way around this.